db/migrate/20210312095649_add_credits_awarded_to_badges.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreditsAwardedToBadges < ActiveRecord::Migration[6.0]
+  def change
+    add_column :badges, :credits_awarded, :integer, default: 0
```

Comment:
If we have `default: 0` for the credits awarded on badges, should we also have `null: false`? Is there any reason we _would_ want to allow `null` if we have a default here? 🤔

Reply:
Big +1 on this. It's something I forget pretty regularly, too.